library/ReplicatedTable.cs

```diff
@@ -365,13 +270,13 @@ private TableOperationType GetOpType(TableOperation operation)
                     {
                         // Set appropriate values in ETag and _rtable_Version for merge, delete, replace,
                         // insertormerge, insertorreplace
-                        row.ETag = retrievedResult.Etag;
+                        row.ETag = new ETag(retrievedResult.Etag);
                         row._rtable_Version = currentRow._rtable_Version + 1;
                     }
                     else
                     {
                         // Initialize Etag if the row is not present
-                        row.ETag = null;
+                        row.ETag = ETag.All;
```

Comment:
See if this needs to be new etag vs Etag.All

Reply:
new ETag() is same as default